discordetf/decoder.go

```diff
@@ -260,7 +259,7 @@ func (d *decoder) readMapWithIDIntoSlice() (int64, []byte, error) {
 
 	err := d.checkByte(ettMap)
 	if err != nil {
-		return 0, nil, errors.WithStack(err)
+		return 0, nil, xerrors.Errorf(": %w", err)
```

Comment:
Usually I put a bit of context into the error before the wrap like so

```suggestion
		return 0, nil, xerrors.Errorf("failed to verify map byte: %w", err)
```


Though, the `failed to` part is extremely repetitive. At work I've started trying just

```suggestion
		return 0, nil, xerrors.Errorf("verify map byte: %w", err)
```

I haven't worked much on this since I started using the new format at work, but I think I'd to prefer to use the latter now.

Thoughts?

Reply:
The reason I didn't put a context was because of the ``WithStack`` use originally, so I thought maybe context wasn't wanted here.

The new format looks good to me and yes I agree ``failed`` to can get repetitive

Reply:
When I wrote this I think I was just too lazy to do a full wrap, so i just did a `WithStack`. If we're fixing it now I'd like to fix my laziness as well :)

Reply:
😄